engine-comparison/runner.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+. metadata.cfg
+. parameters.cfg
+. fengine.cfg
```

Comment:
Yeah, for the runner, every file begins in just one directory, which is the CWD for `runner.sh` (inside the container)